frontend/src/lib/components/InsightCard/InsightCard.tsx

```diff
@@ -27,6 +28,8 @@ export interface InsightCardProps extends React.HTMLAttributes<HTMLDivElement> {
     apiError: boolean
     /** Whether the card should be highlighted. */
     highlighted: boolean
+    /** Whether resize handles should be shown. */
+    resizable: boolean
```

Comment:
non-blocking nit that slightly more descriptive variable name would remove the need for the comment :)

Reply:
👍 That's fair